Add `next-multilingual` example (#29386)
vercel/next.js: examples/with-next-multilingual/README.md

# with-next-multilingual - internationalization using next-multilingual

> The `next-multilingual` module is a unique i18n module for Next.js that offers localized URLs and modular string management on top of other standard features required for multilingual applications.

Reviewer: Hey! Could you add the repository here instead of linking out? Otherwise, `create-next-app -e with-next-multilingual` won't work.

Author: Do you mean adding the entire https://github.com/Avansai/next-multilingual-example repo? Does this mean I will have to keep it in sync?

Reviewer: We'd prefer that, but I won't block this for it.

Author: Thanks - let me see if I can find a way to semi-automate that into our pipeline and maybe do a follow-up PR.

> # with-next-multilingual - internationalization using next-multilingual
> 
> The `next-multilingual` module is a unique i18n module for Next.js that offers localized URLs and modular string management on top of other standard features required for multilingual applications.
> 
> See our solution in action on our [example repository](https://github.com/Avansai/next-multilingual-example).
> 
> Or visit our [main repository](https://github.com/Avansai/next-multilingual) to learn more about how `next-multilingual` can help you apply i18n best practices with its rich feature set.

Reviewer: ```suggestion
# with-next-multilingual

The `next-multilingual` module is a unique i18n module for Next.js that offers localized URLs and modular string management on top of other standard features required for multilingual applications.

[View the example](https://github.com/Avansai/next-multilingual-example) or the [main repository](https://github.com/Avansai/next-multilingual).
```

Reviewer: Small changes, good to go!
